Licensing: Update enterprise documentation (#78276)
grafana/grafana: docs/sources/administration/enterprise-licensing/_index.md

### Step 2. Add your license to a Grafana instance

You must install a Grafana Enterprise build to use the enterprise features, which you can [download](https://grafana.com/grafana/download?edition=enterprise).

{{% admonition type="note" %}}

If you already use Grafana OSS, you can replace it with the same version of Grafana Enterprise.

> It is recommended that you back up the configuration and database before switching.

Reviewer: ```suggestion
Ensure that you back up the configuration and database before proceeding.
```
